cedar-policy-core/src/ast/policy_set.rs

```diff
@@ -87,10 +87,13 @@ impl From<PolicySet> for LiteralPolicySet {
 /// Potential errors when working with `PolicySet`s.
 #[derive(Error, Debug)]
 pub enum PolicySetError {
-    /// There was a `PolicyId` collision in either the set of templates or the set of policies.
-    /// Nothing was added.
-    #[error("collision in policy id")]
-    Occupied,
+    /// There was a duplicate `PolicyId` encountered in either the set of
+    /// templates or the set of policies.
+    #[error("duplicate template or policy id: {id}")]
+    Occupied {
+        /// `PolicyId` that was duplicate
```

Comment:
```suggestion
        /// [`PolicyId`] that was duplicate
```

Reply:
Thanks for the suggestion, this actually needs to be `PolicyID` since we're in Core